[YUNIKORN-2354] Detailed queue info

doupache opened the pull request:
### What is this PR for?
<img width="1195" alt="queue-info" src="https://github.com/user-attachments/assets/8db6db82-0805-4465-b41f-3a343fd33aed">


https://github.com/user-attachments/assets/11365d21-50bc-4003-938c-6020a136d8e8



### What type of PR is it?
* [x] - Feature

### What is the Jira issue?
[YUNIKORN-2354](https://issues.apache.org/jira/browse/YUNIKORN-2354)

chenyulin0719 commented on src/app/utils/common.util.ts:
```diff
@@ -81,8 +81,42 @@ export class CommonUtil {
+      return '<strong>Memory:</strong> n/a';
+    }
+    let memory = value.split('%')[0] + '%';
+    console.log('mem', memory); 
```
The console log should be removed.

chenyulin0719 reviewed (changes requested):
@doupache Could you also add tests for below functions?

- absoluteUsedMemoryColumnFormatter()
- absoluteUsedCPUColumnFormatter()
- resourceColumnFormatter()

doupache commented:
Thanks @chenyulin0719 for reviewing!

I have added the formatter tests and also removed the unnecessary console logs. 
Also added the missing guarantee block.

chenyulin0719 reviewed (changes requested):
Hi @doupache,

Thanks for the update.
I found that this PR breaks the resource format on the application page. Could you check it?

<img width="1463" alt="image" src="https://github.com/user-attachments/assets/758cca5e-85eb-4c6c-9f3f-640a4170fe0a">

doupache commented:
I duplicated the `ResourceColumnFormatter`, creating one version for the queue page and another for the application page. I believe this duplication is acceptable and makes the code more readable.

<img width="1253" alt="image" src="https://github.com/user-attachments/assets/fe1b0979-0a45-471c-a9dc-05865a7f5cf2">

doupache commented:
### Inactive queue 
<img width="1170" alt="image" src="https://github.com/user-attachments/assets/7f7b889e-7dbd-416a-9e62-d372de232b29">

### Active queue
<img width="1197" alt="image" src="https://github.com/user-attachments/assets/c79bd39c-ddea-4ad5-b2a1-274e2c29dc01">


@craigcondit @chenyulin0719  I  have made layout tighter and add new queue data to `json.db`. 
PTLA

ryankert01 reviewed (approved):
+1, it's compact now! nice.

craigcondit reviewed (approved):
+1 LGTM.
